common/css/devdocs.css

```diff
@@ -1 +1 @@
-.home-contributors{margin-bottom:2em}.message-banner{margin:0}redoc[spec-url] table{table-layout:auto}redoc[spec-url] .menu-subitems .menu-item-title{text-transform:none}.language-bash.highlighter-rouge code:before{display:inline;content:'$';margin-right:5px;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}.page-updated{margin:0 0 10px 0;color:#767676;cursor:help}.page-updated img{width:18px;height:18px}
+.home-contributors{margin-bottom:2em}.message-banner{margin:0}redoc[spec-url] table{table-layout:fixed}redoc[spec-url] .menu-subitems .menu-item-title{text-transform:none}.language-bash.highlighter-rouge code:before{display:inline;content:'$';margin-right:5px;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}.page-updated{margin:0 0 10px 0;color:#767676;cursor:help}.page-updated img{width:18px;height:18px}.webapitable{table-layout:fixed}
```

Comment:
Maybe table should remain with `table-layout:auto`

Reply:
Do not change this file directly, please. This file is generated from `scss/devdocs.scss`. To generate it, use `npm run build:css`

Reply:
Hi @belbiy 

Thanks for your suggestion. I have one confusion in  https://github.com/magento-devdocs/devdocs-theme Project.

Do I need to fork clone this repo in https://github.com/hitesh-wagento/devdocs or in new place?

Thanks for advance